okapi-core/src/test/java/org/folio/okapi/util/ModuleUtilTest.java

```diff
@@ -154,4 +154,64 @@ void testModuleList() {
     list.add(md);
     assertThat(ModuleUtil.moduleList(list)).isEqualTo("foo-1.0.0, bar-2.0.0");
   }
+
+  @Test
+  void testGetObsolete1() {
+    assertThat(modulesList.size()).isEqualTo(104);
+    List<ModuleDescriptor> obsolete = ModuleUtil.getObsolete(modulesList);
+    assertThat(obsolete.isEmpty()).isTrue();
+  }
+
+  @Test
+  void testGetObsoleteBackend() {
+    String ids[] = new String [] {
+        "mod-a-0.9.0-SNAPSHOT.3",
+        "mod-a-1.0.0-SNAPSHOT.4",
+        "mod-a-1.0.0",
+        "mod-b-1.0.0-SNAPSHOT.1",
+        "mod-a-1.1.0-SNAPSHOT.5",
+        "mod-a-0.8.0-SNAPSHOT.1",
+        "mod-a-0.8.0-SNAPSHOT.2",
+        "mod-a-0.8.0",
+    };
+    List<ModuleDescriptor> mds = new LinkedList<>();
+    for (int i = 0; i < ids.length; i++) {
+      ModuleDescriptor md = new ModuleDescriptor();
+      md.setId(ids[i]);
+      mds.add(md);
+    }
+    assertThat(mds.size()).isEqualTo(8);
```

Comment:
This assert is not needed. It doesn't test the class under test but our test setup.